### ISSUE TITLE
Arguments in the doc string of `create_experiment` don't match the real arguments

### ISSUE THREAD

**Issue by yaelbh** [Qiskit-Extensions/qiskit-ibm-experiment#52]
I'd expect the linter to catch it

**Comment by yaelbh** [comment]
Same also for `_experiment_data_to_api`

**Comment by gadial** [comment]
> I'd expect the linter to catch it

You are correct, I assumed this as well. But now I see this is an extension of pylint I did not enable here:

```
load-plugins=pylint.extensions.docparams, # enable checking of docstring args
    pylint.extensions.docstyle,  # basic docstring style checks
```

This will be fixed as part of #53 .

**Comment by yaelbh** [comment]
Closing because this was apparently fixed in #53
